lisa/analysis/tasks.py

```diff
@@ -632,4 +677,50 @@ def plot_tasks_forks_heatmap(self, xbins=100, colormap=None, filepath=None):
 
         return axis
 
+    @df_task_activation.used_events
+    def plot_task_activation(self, task, filepath=None, axis=None,
+            alpha=None, overlay=False, **kwargs):
+        """
+        Plot task activations, in a style similar to kernelshark.
+
+        :param task: the task to report activations of
+        :type task: int or str
+
+        :param alpha: transparency level of the plot.
+        :type task: float
+
+        :param overlay: If ``True``, assumes that ``axis`` already contains a
+            plot, so it will adjust automatically the height and transparency of
+            the plot to blend with existing data.
+        :type task: bool
+
+        :param kwargs: Keyword arguments passed to :meth:`df_task_activation`.
```

Comment:
Nit: I've used `:Keyword arguments:` elsewhere, but I think some `kwargs` have slipped here and there anyway...

Reply:
yes, I've never used that definition list, I'll batch convert all the `:param kwargs:` in a separate PR.